Bonsai.Templates/Bonsai.Templates.csproj

```
@@ -2,20 +2,31 @@
 
   <PropertyGroup>
     <PackageType>Template</PackageType>
-    <PackageVersion>2.8.2</PackageVersion>
     <PackageId>Bonsai.Templates</PackageId>
     <Title>Bonsai Templates</Title>
     <Authors>Bonsai</Authors>
     <Description>Templates for creating a C# Bonsai package (.nupkg)</Description>
     <PackageTags>Bonsai Rx Package Environment Templates</PackageTags>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeContentInPack>true</IncludeContentInPack>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
     <ContentTargetFolders>content</ContentTargetFolders>
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <PackageLicenseUrl></PackageLicenseUrl>
     <PackageIcon>icon.png</PackageIcon>
     <PackageIconUrl></PackageIconUrl>
+
+    <!-- This package only contains .NET templates and does not contain any code to be packed -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>false</IncludeSymbols>
+
+    <!-- Ignore warnings about Setup.ps1 as it is not meant to be a NuGet setup script -->
+    <NoWarn>$(NoWarn);NU5110;NU5111</NoWarn>
+
+    <!--
+      This warning basically happens because this package has no assemblies but does have dependencies
+      https://github.com/NuGet/Home/issues/8583
```

```suggestion
      This warning basically happens because this package has no assemblies but does have dependencies
      https://github.com/NuGet/Home/issues/8583
```
```suggestion
      This warning happens because this package has no assemblies
      https://github.com/NuGet/Home/issues/8583
```

I guess in this case it doesn't even have dependencies as it is just a template package.

---

Good catch, but I left the word "basically" since the warning doesn't literally happen because there's no assembly, it's just the first domino which triggers a chain reaction which causes that warning.

When you encounter the warning in the wild you'd never guess it was due to there being no assembly in the package:

> warning NU5128: Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the other location.